Where_Is_My_Wife/Assets/_Main/Scripts/Stage/LavaRise.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class LavaRise : MonoBehaviour
```

Comment:
I remember chambers where the lava needs to stop, where is that code?

Reply:
already added it, it's called lavaTrigger